images/xr-standard-mapping.svg

```diff
@@ -688,17 +688,19 @@
          x="103.03984"
          y="283.68451"
          style="font-size:4.23333311px;line-height:0.5;text-align:center;text-anchor:middle;fill:#ff0909;fill-opacity:1;stroke-width:0.26458332"
-         id="tspan5165-0-0-0"><tspan
+         id="tspan1063"><tspan
    style="fill:#000000;fill-opacity:1"
-   id="tspan5533">NOTE: </tspan>buttons[3]<tspan
+   id="tspan5533">NOTE: For this device <tspan
+   style="fill:#0000ff;fill-opacity:1"
+   id="tspan1066">axes[0]</tspan>, <tspan
+   style="fill:#0000ff;fill-opacity:1"
+   id="tspan1078">axes[1]</tspan>, and </tspan>buttons[1]<tspan
    style="fill:#000000;fill-opacity:1"
-   id="tspan5539"> is left empty, since it is reserved</tspan></tspan><tspan
+   id="tspan5539"> are</tspan></tspan><tspan
          sodipodi:role="line"
          x="103.03984"
          y="289.21323"
-         style="font-size:4.23333311px;line-height:0.5;text-align:center;text-anchor:middle;fill:#ff0909;fill-opacity:1;stroke-width:0.26458332"
-         id="tspan5531"><tspan
-           style="fill:#000000;fill-opacity:1"
-           id="tspan5541">for a secondary joystick or touchpad click</tspan></tspan></text>
+         style="font-size:4.23333311px;line-height:0.5;text-align:center;text-anchor:middle;fill:#000000;fill-opacity:1;stroke-width:0.26458332"
+         id="tspan5531">placeholders, since they are reserved for Touchpad inputs</tspan></text>
   </g>
 </svg>
```

Comment:
Not sure where to leave this comment, but can we make sure to explicitly state that the second image is an *example* advanced controller?

Reply:
It's the text on the image I was hoping to change....

Reply:
Done.